File: 9. JS Front-End/Conditional Statements/GodzillaVsKong.js
function godzillaVsKong(input) {

    let movieBudget = Number(input[0]);
    let extras = Number(input[1]);
    let extraxClothesPrice = Number(input[2]);
    
    let decoPrice = movieBudget * 0.10;

    let moviePrice;

    if (extras > 150) {
        let clothesPrice = extras * extraxClothesPrice;

        let clothesPriceWithDiscount = clothesPrice - (clothesPrice * 0.10);

        moviePrice = decoPrice + clothesPriceWithDiscount;

    } else {
        let clothesPrice = extras * extraxClothesPrice;
        moviePrice = decoPrice + clothesPrice;
    }
    

    if (moviePrice <= movieBudget) {
        console.log("Action!");
        console.log(`Wingard starts filming with ${(movieBudget - moviePrice).toFixed(2)} leva left.`);
    } else {
        console.log("Not enough money!");
        console.log(`Wingard needs ${(moviePrice - movieBudget).toFixed(2)} leva more.`)
    }

}

godzillaVsKong(["9587.88",
"222",
"55.68"])

File: Programming basics with Java/IF & ELSE Statements/GodzillaVsKong.java
import java.util.Scanner;

public class GodzillaVsKong {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        double budget = Double.parseDouble(scanner.nextLine());
        int extras = Integer.parseInt(scanner.nextLine()); // how many extras we have
        double clothingPrice = Double.parseDouble(scanner.nextLine());

        double discount = 0.0;

        double decor = budget * (10.0f/100.f);
        double price = extras * clothingPrice; // price for the clothing

        if (extras > 150){
            discount = price * (10.0f/100.0f);
            price -= discount;
        }

        double totalPrice = decor + price;

        if (totalPrice > budget){
            System.out.println("Not enough money!");
            System.out.printf("Wingard needs %.02f leva more.", totalPrice - budget);
        }else {
            System.out.println("Action!");
            System.out.printf("Wingard starts filming with %.02f leva left.", budget- totalPrice);
        }

    }
}



